try {
  throw "we broke this app!";
  alert("we tried");
} catch (error) {
  console.error(error);
}
